.github/workflows/sycl-linux-build.yml

```diff
@@ -296,6 +296,7 @@ jobs:
         cmake --build $GITHUB_WORKSPACE/build --target install-llvm-cov
         cmake --build $GITHUB_WORKSPACE/build --target install-llvm-profdata
         cmake --build $GITHUB_WORKSPACE/build --target install-compiler-rt
+        cmake --install $GITHUB_WORKSPACE/build --component DeviceConfigFile
```

Comment:
basically we only install this file in CI now right?

Reply:
With this patch - yes. It's the initial intention as I understand:
<img width="949" height="159" alt="image" src="https://github.com/user-attachments/assets/c1216112-2848-4feb-8d6a-1311c369bfea" />

Reply:
I think this needs a comment in the code about what the file does and why we do it here.